app/views/events/_my_schedule.html.erb

```diff
@@ -0,0 +1,115 @@
+<%
+    # Consider moving all of this to a helper
+    # Also move this to a partial, so that we can include it on the Events page.
+    sessions = event.sessions
+    sessions = sessions.sort
+    table_sessions = {}
+
+    sessions.each do |session|
+      table_sessions[session] = nil
+    end
+
+    user_event.registration_tables.each do |reg_table|
+      session = reg_table.table.session
+      table_sessions[session] = reg_table
+    end
+    user_event.game_masters.each do |game_master|
+      session = game_master.table.session
+      table_sessions[session] = game_master
+    end
+%>
+
+<div class="panel panel-padded" style="border: solid thin black;">
+  <% if event.online? %>
+    <h3>My <%= event.name %> Schedule</h3>
+    <!-- Add some padding on the elements below-->
+    <div class="bg-warning top-buffer">
+      <div>
+        <span style="font-size: 1.5em" class="glyphicon glyphicon-bullhorn"></span>
+        <%= event.name %> Chat Server:
+        <a target="_blank" href="<%= @event.chat_server_url %>"><%= @event.chat_server %></a>
+      </div>
+    </div>
+
+    <div class="bg-warning top-buffer">
+      <div><span style="font-size: 1.5em" class="glyphicon glyphicon-info-sign"></span>
+        If your game location is 'online', please click on the 'View Table Details' link for that session. Your game
+        master will have important information there, such as their <abbr title="Virtual Table Top">VTT</abbr> link, and
+        which chat room the game will be using. Instructions on how to sign-in to the table will be there as well.
+      </div>
```

Comment:
Suggested rewording: Table details for online games have important information from your GM, including your Virtual Table Top (VTT) link and game chatroom. To view, click on the "View Table Details" button.